week09/movie/src/pages/DetailPage.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import axios from "axios";
+import Detail from "../components/Detail";
+import Loader from "../components/Loader";
+
+const Container = styled.div`
+    display: flex;
+    justify-content: center;
+`;
+
+const DetailPage = () => {
+    const { id } = useParams();
+    const [movie, setMovie] = useState(null);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        async function getDetail() {
+            try {
+                const res = await axios.get(
+                    `https://yts.mx/api/v2/movie_details.json?movie_id=${id}`
+                );
+                setMovie(res.data.data.movie);
+                setLoading(false);
+                console.log(movie);
+            } catch (error) {
+                console.log(error);
+            }
+        }
+        getDetail();
+    }, [id]);
+
```

Comment:
- [x] 3. 종속성 배열 movie
```suggestion
  useEffect(() => {
    console.log(movie);
  }, [movie]);
```
분리한 useEffect 훅에서 `console.log(movie);`를 출력하고 movie를 의존성 배열로 넣어 상태를 관리합니다.